controllers:
  robot_state_controller:
    type: robot_state_controller/RobotStateController
    publish_rate: 50
  imu_sensor_controller:
    type: imu_sensor_controller/ImuSensorController
    publish_rate: 100
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50
  chassis_balance_controller:
    type: rm_chassis_controllers/BalanceController
    joint_left_name: "left_wheel_joint"
    joint_right_name: "right_wheel_joint"
    wheel_radius: 0.06
    wheel_base: 0.405
    com_pitch_offset: -0.045
    a: [ [ 0, 1, 0, 0 ],
         [ 432.8077, 0, 0, 0 ],
         [ 73.9215, 0, 0, 0 ],
         [ 0, 0, 0, 0 ] ]
    b: [ [ 0, 0 ],
         [ -49.6930, -49.6930 ],
         [ 7.3620, 7.3620 ],
         [ -26.3737, 26.3737 ] ]
    q: [ [ 1, 0, 0, 0 ],
         [ 0, 1, 0, 0 ],
         [ 0, 0, 500, 0 ],
         [ 0, 0, 0, 1 ] ]
    r: [ [ 1, 0 ],
         [ 0, 1 ] ]
    twist_covariance_diagonal: [ 0.001, 0.001, 0.001, 0.001, 0.001, 0.001 ]
